lib/src/web/web_io.dart

```diff
@@ -88,37 +89,54 @@ class WebFileSystemProvider implements MultiFileProvider {
     return _ioFolderAdapter.fromIOFiles(files);
   }
 
+  Future<IOFile> _platformFileToStreamFile(PlatformFile platformFile) {
+    return _ioFileAdapter.fromReadStreamGenerator(
+      platformFile.readStream!,
+      platformFile.size,
+      name: platformFile.name,
+      lastModifiedDate: platformFile.lastModified ?? DateTime.now(),
```

Comment:
For now, let's just use `DateTime.now()` as it was before.
`platformFile.lastModified` breaks on Firefox and we need to be consistent between browsers and platforms.

Reply:
Sure thing